kaggle_environments/envs/mab/mab.py

```diff
@@ -99,11 +99,11 @@ def sample():
     action_histogram = kaggle_environments.helpers.histogram(shared_observation.last_actions)
 
     for index, threshold in enumerate(thresholds):
-        # Every time a threshold is selected it is multiplied by (1 + decay_rate) for each agent that selected it.
-        # When a threshold is not selected it is reduced by (1 + decay_rate) ^ -1.
-        action_count = action_histogram[index] if index in action_histogram else -1
-        update_rate = (1 + configuration.decay_rate) ** action_count
-        thresholds[index] = max(threshold * update_rate, initial_thresholds[index])
+        # Every time a threshold is selected it is multiplied by (decay_rate) for each agent that selected it.
+        # When a threshold is not selected it is reduced by (decay_rate) ^ 0 (i.e. no recovery).
+        action_count = action_histogram[index] if index in action_histogram else 0
+        update_rate = (configuration.decay_rate) ** action_count
```

Comment:
You'll need to also update mab.json to change the default values for params from .03 to .97, e.g.